src/main/java/org/jabref/model/groups/SearchGroup.java

```diff
@@ -15,28 +19,60 @@
  * multiple fields.
  */
 public class SearchGroup extends AbstractGroup {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(SearchGroup.class);
-    private final GroupSearchQuery query;
+
+    @ADR(38)
```

Comment:
In DuplicateSearch it uses the comment format:
![alt text](grafik.png)

Reply:
In Java, annotations are limited. We tried to use [`e-adr`](https://adr.github.io/e-adr/) whereever possible. Where not, we used Java comments. #research.